### PR TITLE
Jasminum sambac

### PR DESCRIPTION
**Pull request by philippfe**
https://github.com/instructlab/taxonomy/issues/835

If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/science/botany/jasminum/qna.txt
Comment (jjasghar) [inline knowledge/science/botany/jasminum/qna.txt:1]: Please change this file name to `qna.yaml`

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the file name issue, then we can run the linter.

File: knowledge/science/botany/jasminum/qna.txt
Comment (philippfe) [reply knowledge/science/botany/jasminum/qna.txt:1]: very sorry - I renamed the file

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issue.

**Comment by philippfe** [comment]
Hi, I fixed the issues - the only I thing the linter mentioned which I couldn't identify is the error in line 6, stating a missing ":". In my opinion everything looks fine and should work, similar to the other questions.
Please let me know, if I'm missing something.
Thanks for your help and patience!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolve the linting issues.

**Comment by philippfe** [comment]
Linting issues should be fixed now.
Kind regards,
Philipp

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please check here for the linting issues, you have a few still hanging around :)

**Comment by philippfe** [comment]
Hm, yes it seems, I have added new issues, sorry - will fix it asap

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues listed here: https://github.com/instructlab/taxonomy/pull/954/files

**Comment by jjasghar** [comment]
Closed due to the combined PR here: https://github.com/instructlab/taxonomy/pull/1140

**Pull request closed without merging**